types/test.ts

```diff
@@ -127,6 +128,30 @@ export function testConfigCallback() {
   })
 }
 
+export function testInstantiatedStore() {
+  render(SomeComponent, {
+    store: new Vuex.Store({
```

Comment:
I added this test to ensure that a `Store` could be provided as `StoreOptions<S>` and it looks like it works!